content/en/getting_started/integrations.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Introduction to Integrations
+kind: documentation
+further_reading:
+- link: "https://learn.datadoghq.com/"
+  tag: "Learning Center"
+  text: "Introduction to Datadog"
+- link: "/integrations/"
+  tag: "Integrations"
+  text: "Datadog's full list of integrations"
+---
+
+This is a guide for using integrations, for information about building a new integration, see the [Create a new integration][1] page.
+
+An integration, at the highest level, is when you assemble a unified system from units that are usually considered separately. At Datadog, you can use integrations to bring together all of the units of your infrastructure and gain insight into the unified system as a whole—you can see pieces individually and also how individual pieces are impacting the whole.
+
+* Over 250 Agent integrations are supported in the [integrations core][2].
+* Even more Agent integrations have been built by our open source community, in [integrations extras][3].
+* Find all available integrations on the [Integrations page in the app][4].
+* You can build your own integration using the [Datadog API][5].
+* The Agent is [open source][6] and you can [create your own custom check][7].
+
+Datadog provides three main types of integrations:
```

Comment:
I would mention somehow in this paragraph the existence of custom checks, that are similar to an agent-based integration but with substantial differences in terms of complexity and reuse.